Makefile

```diff
@@ -96,8 +96,8 @@ endif
 #
 # Example:
 #   make check-test
-check-test: export KUBE_COVER= -cover -covermode=atomic
-check-test: export KUBE_RACE=  -race
+check-test: export COVERAGE_SPEC=-cover -covermode=atomic
+check-test: export DETECT_RACES=true
```

Comment:
I think it was because we copy-pasted upstream Kubernetes' script at the get-go. Someone can weigh in if this breaks something.